dev_guide/authorization.adoc

```diff
@@ -0,0 +1,83 @@
+[[dev-guide-authorization]]
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[checking-if-users-can-create-pods]]
+== Checking if Users Can Create Pods
+
+Using the `scc-review` and `scc-subject-review` options, you can see if an
+individual user, or a user under a specific service account, can create a pod.
```

Comment:
Let's mention at least once that it check not only whether user can create a pod but also whether he can update a pod.